### ISSUE TITLE
test cases failing due to neighbourhood changes

### ISSUE THREAD

**Issue by missinglink** [pelias/acceptance-tests#256]
ticket to keep track of which cases need to be updated due to the changed made in https://github.com/pelias/wof-pip-service/pull/36

```
✘ regression [4] "{"text":"starbucks 10010"}": score 12 out of 13
  diff:
    neighbourhood
      expected: Midtown South
      actual:   
```

**Comment by orangejulius** [comment]
https://github.com/pelias/wof-pip-service/pull/41 should help fix these, although there may still be some stragglers

**Comment by trescube** [comment]
Yep, [Midtown South](https://whosonfirst.mapzen.com/spelunker/id/85865701/#16/40.7414/-73.9894) has `mz:hierarchy_label` value of `1` which will fix the issue.

**Comment by orangejulius** [comment]
I think all these issues are all fixed, are any still happening?

**Comment by orangejulius** [comment]
The test failures we had from this are:

- [x] Midtown South (neighbourhood value has been removed from expected section)
- [x] Greenwich Village (fixed with pelias/wof-pip-service#41)

They've all been fixed!
